docs/user-guide/safety-security/guardrails.md

```diff
@@ -51,6 +51,90 @@ if response.stop_reason == "guardrail_intervened":
 print(f"Conversation: {json.dumps(agent.messages, indent=4)}")
 ```
 
+Alternatively, if you want to implement your own soft-launching guardrails, you can utilize Hooks along with Bedrock's ApplyGuardrail API in shadow mode. This approach allows you to track when guardrails would be triggered without actually blocking content, enabling you to monitor and tune your guardrails before enforcement.
+
+Steps:
+1. Create a NotifyOnlyGuardrailsHook class that contains hooks
```

Comment:
Can you verify that this renders correctly when served by mkdocs?  IIRC mkdocs doesn't like a list without a blank link - but I could be wrong